### ISSUE TITLE
v0.60.0 broken due to missing django_components.utils module

### ISSUE THREAD

**Issue by kaiku** [EmilStenstrom/django-components#403]
v0.60.0 refuses to load due to a missing `__init__.py` in the newly created `utils` dir introduced in https://github.com/EmilStenstrom/django-components/pull/395, cc @JuroOravec.

Relevant stack trace:
```
    from django_components.utils.autodiscover import search
ModuleNotFoundError: No module named 'django_components.utils'
```

I'm not able to test this now but I think the solution is easy:
1. Add an `__init__.py` to `utils`
2. (Preferred) Rename `utils/autodiscover.py` to `utils.py`. I think for simplicity's sake, everything can and should go in one file for this project.

**Comment by JuroOravec** [comment]
@kaiku Thanks for making an issue for this, I've noticed it too last night, but it was already too late. Working on an MR for this atm.

**Comment by EmilStenstrom** [comment]
@kaiku Thanks for reporting, a fix for this is released as 0.61 in pypi. Please update!
